test/unit/bps_tree_with_child_cards.cc

```diff
@@ -0,0 +1,14 @@
+/* Enable child_cards array for each tree tested. */
+#define BPS_BLOCK_CHILD_CARDS_ARRAY
+/*
+ * Some branches lead to dummy rebalancing (moving data of zero size) when
+ * the block size is small because of integer arithmetic on reballancing.
+ * This raises assertions, because some data movement routines are designed
+ * to only move non-zero amount of data. Let's make the block size greater
+ * to prevent this.
+ */
+#define SMALL_BLOCK_SIZE 256
+#define COMPARE_WITH_SPTREE_CHECK_BRANCHES_ELEM_LIMIT 2048
+#define TEST_COUNT_IN_LEAF 30
+#define TEST_COUNT_IN_INNER 12
```

Comment:
Let's introduce the new public API function that would return the number of BPS tree elements less than the specified key to test the feature. It can be useful anyway.

Reply:
Introduced `find_get_offset` and `[lower|upper]_bound[_elem]_get_offset` families of functions.